src/main/java/ddingdong/ddingdongBE/domain/form/service/FacadeCentralFormServiceImpl.java

```diff
@@ -68,9 +69,12 @@ public void createForm(CreateFormCommand createFormCommand) {
 
         Form form = createFormCommand.toEntity(club);
         Form savedForm = formService.create(form);
-
-        List<FormField> formFields = toCreateFormFields(savedForm,
-                createFormCommand.formFieldCommands());
+        List<FormField> formFields;
+        if (savedForm.getSections().size() > 1) {
```

Comment:
p4) form 내부에서 처리해도 될 로직이네요!
```suggestion
        if (savedForm.isLargerSectionThan(1)) {
```

1도 상수처리 해주면 좋을 것 같아요~!

Reply:
상수처리 완료했습니다!